server/testdriver/junit5/src/main/java/org/infinispan/server/test/junit5/InfispanServerExtensionBuilder.java

```diff
@@ -0,0 +1,43 @@
+package org.infinispan.server.test.junit5;
+
+import org.infinispan.server.test.core.AbstractServerConfigBuilder;
+import org.infinispan.server.test.core.ServerRunMode;
+
+import static org.infinispan.server.test.core.AbstractInfinispanServerDriver.DEFAULT_CLUSTERED_INFINISPAN_CONFIG_FILE_NAME;
+
+/**
+ * Infinispan Server Extension Builder
```

Comment:
Shouldn't we have more info about how to use the extension, or at least a `@see InfinispanServerExtension`?

Reply:
Dan,  possibly, but not in this PR please. I can create a JIRA to document and improve after this gets in.

Reply:
ok for the javadoc in the other file